lib/Wikia/src/Logger/LogstashFormatter.php

```diff
@@ -12,15 +12,19 @@
 
 
 class LogstashFormatter extends \Monolog\Formatter\LogstashFormatter {
-	public function __construct()
-	{
-		// prevent "Undefined variable: applicationName" notice
-		parent::__construct(null);
+
+	const APPNAME = 'mediawiki';
+
+	/**
+	 * @param string $appname
+	 */
+	public function __construct($appname = self::APPNAME) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method "__construct" to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1185)